index.js

```diff
@@ -76,6 +76,13 @@ module.exports = {
     this.jshintrc = app.options.jshintrc;
   },
 
+  postBuild: function () {
```

Comment:
Also in init with a `process.exit(1);` causes it to not be able to be installed in the first place.